migrations.js

```diff
@@ -79,15 +121,18 @@ var runMigrations = function(meta, migrations, callback) {
 
 module.exports = {
   run: function(callback) {
-    getMeta(function(err, meta) {
+    module.exports.get(function(err, migrations) {
       if (err) {
         return callback(err);
       }
-      module.exports.get(function(err, migrations) {
+      if (!migrations.length) {
```

Comment:
Is this an alternative to checking `if(err)`?  Can't see why the migrations list would ever be falsy, otherwise.